lib/node_loader.js

```diff
@@ -59,7 +62,19 @@ AWS.util.update(AWS.Config.prototype.keys, {
     return new AWS.CredentialProviderChain();
   },
   region: function() {
-    return process.env.AWS_REGION || process.env.AMAZON_REGION;
+    var env = process.env;
+    var region = env.AWS_REGION || env.AMAZON_REGION;
+    if (!region && env.AWS_SDK_LOAD_CONFIG) {
```

Comment:
I believe we're actually supposed to check the credentials file for region, then the config file, similar to the CLI/boto.